src/schedule.jl

```diff
@@ -96,12 +96,19 @@ abbrev(x::JuliaConTalkType) = abbrev(typeof(x))
 is_schedule_json_available() = isfile(joinpath(CACHE_DIR, "schedule.json"))
 
 """
+    get_conference_schedule(; speaker=nothing)
+
 Get the conference schedule as a DataFrame.
 On first call, the schedule is downloaded from Pretalx and cached for further usage.
+`speaker` can be a name or a string identifying the speaker to filter the schedule.
 """
-function get_conference_schedule()
+function get_conference_schedule(; speaker=nothing)
     isassigned(jcon) || update_schedule()
-    return jcon[]
+    # filter for speaker
+    jcon_filt = filter(jcon[]; view=true) do talk
```

Comment:
This, I believe, will change the return type of `get_conference_schedule` since `filter` does return a `SubDataFrame` or something like that if I'm not mistaken.

Reply:
Also I think it is the "wrong" place for the filtering (see below).

Reply:
If we remove the `view` it is the same type, isn't it?

I see your concerns, should I move it to the `today` filter where we filter for date as well?